src/02-common-principles.md

```diff
@@ -559,6 +570,11 @@ Describing dates and timestamps:
     within each subject to maintain the interval information.
     For example: `1867-06-15T13:45:30`
 
+-   WARNING: The Neuromag/Elekta/MEGIN file format for MEG (`.fif`) does *not*
+    support recording dates earlier than `1902` roughly.
+    See [MEG-file-formats](./99-appendices/06-meg-file-formats.md#recording-dates-in-fif-files)
```

Comment:
I think you might want to state that, even if it's another format, certain software packages (such as MNE) support only fif based writing. Thus, even if you read from a non-fif software, but use MNE to process the data, you might still hit this issue.

Reply:
good point, thanks. Added:

>     Some analysis software packages (e.g., MNE-Python) handle their data as `.fif`
    internally and will break if recording dates are specified prior to `1902`,
    even if the original format is not `.fif`.